study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/StudyBuilderCli.java

```diff
@@ -245,4 +258,32 @@ private void runInvalidateStudy(Config appCfg, Config studyCfg, StudyBuilder bui
         execute(builder::runInvalidate, isDryRun);
         log("done");
     }
+
+    private void runEmails(CommandLine cmd, Path cfgPath, Config studyCfg, Config varsCfg) {
+        var emailBuilder = new EmailBuilder(cfgPath, studyCfg, varsCfg);
+        Set<String> keys = null;
+        if (cmd.hasOption(OPT_EMAIL_KEYS)) {
+            keys = Set.of(cmd.getOptionValue(OPT_EMAIL_KEYS).split(","));
+        }
+
+        if (cmd.hasOption(OPT_CREATE_EMAILS)) {
+            if (keys == null) {
+                log("executing creation of all study sendgrid emails...");
+                emailBuilder.createAll();
```

Comment:
Maybe push `log()` calls down into `emailBuilder` methods?

Reply:
Either way should be fine, but seems to fit better as a log in the CLI, so leaving for now.